### ISSUE TITLE
Wi-Fi Issues on OpenCore (DW1560)

### ISSUE THREAD

**Issue by LeLunZ** [jardenliu/XPS15-9560-Ventura#178]
I changed my Clover System to OpenCore.

Everything works. Only Problem: 

My Wi-Fi is always turned off on start. Then I must activate it. Here it doesn't work. When deactivating and reactivating it, it does work.

Could it be that the non-DW1830 method needs an update?

**Comment by LeLunZ** [comment]
I added the aml from here.

https://github.com/jardenliu/XPS15-9560-Catalina/tree/OpenCore/Post-install/non-DW1830BT(非DW1830使用)

After future testing:
- Bluetooth does work. But AirPods doesn't work. They are shown as connected but you audio is playing over Laptop
- Wi-Fi problem still exists

Also bpr_probedelay (boot args) is removed in BrcmPatchRAM3.
This is from original GitHub Repo:

> bpr_probedelay: Changes mProbeDelay (removed in BrcmPatchRAM3), the delay in ms before probing the device. Default value is 0.

**Comment by LeLunZ** [comment]
I now also tried replacing the USBPorts.kext from there. Also doesn't work.

**Comment by LeLunZ** [comment]
Also tried removing all delays from boot args. Doesn't change anything.

Tried to update from 0.6.1 to 0.6.3.
Changes need to be done:
- BT4LEContinuityFixup.kext is now a Quirck and is not supported through kext

Why do we use HWPEnabler.kext? Isn't it kinda useless, we already got native power management.

Also CodecCommander is kind of unsupported, it seems like the community is using ComboJack...
https://github.com/hackintosh-stuff/ComboJack
